src/main/resources/db/migration/V1.0.0__initial_schema.sql

```diff
@@ -0,0 +1,451 @@
+    created_timestamp timestamp not null
+);
+--TODO?: index by review_body_created_timestamp
+
+
+create type review_point_type as enum ('ADVANTAGE', 'DISADVANTAGE');
+
+create table review_point
+(
+    review_id uuid references review,
+    ordering  int not null check (ordering >= 0),
+    type      review_point_type not null,
+    content   varchar(256) not null,
+    primary key (review_id, ordering)
+);
+-- no sense in creating index by ordering because it does not guarantee select order
+
+create table review_image
+(
+    id        uuid primary key,
+    review_id uuid not null references review,
+    ordering  integer not null check (ordering >= 0)
+);
+-- no sense in creating index by ordering because it does not guarantee select order
+
+create type vote_type as enum ('UP', 'DOWN');
+
+create table review_vote
+(
+    review_id   uuid references review,
+    reviewer_id uuid references actor,
+    type        vote_type not null,
+    primary key (review_id, reviewer_id)
+);
+
+create table review_comment
+(
+    id                uuid primary key,
+    review_id         uuid not null references review,
+    author_id         uuid not null references actor,
+    content           varchar not null,
+    created_timestamp timestamp not null,
+    --TODO: last_modified_timestamp timestamp,
+    is_shown          boolean not null,
+    upvotes_count      integer not null check (upvotes_count >= 0),
+    downvotes_count    integer not null check (downvotes_count >= 0)
+);
+
+create table review_comment_vote
+(
+    review_comment_id uuid references review_comment,
+    reviewer_id       uuid references actor,
+    type              vote_type not null,
+    primary key (review_comment_id, reviewer_id)
+);
+
+create type report_issuer_type as enum ('GUEST', 'REVIEWER', 'BRAND_OWNER', 'SYSTEM');
+
+create type report_status as enum ('OPEN', 'IN_PROGRESS', 'CLOSED');
+
+create table moderator_report_reason
+(
+    id         uuid primary key,
+    name       varchar(128) not null,
+    definition varchar(256) not null,
+    is_shown   boolean not null
+);
+
+create table moderator_report
+(
+    id                      uuid primary key,
+    issuer_id               uuid references actor,
+    status                  report_status not null,
+    assignee_moderator_id   uuid references actor,
+    created_timestamp       timestamp not null,
+    last_modified_timestamp timestamp
+);
+create index moderator_report_created_timestamp_idx ON moderator_report (created_timestamp);
+
+
+create table moderator_report_to_reason
+(
+    report_id uuid references moderator_report,
+    reason_id uuid references moderator_report_reason,
+    primary key (report_id, reason_id)
+);
+
+create table moderator_report_to_review
+(
+    report_id uuid references moderator_report,
+    review_id uuid references review,
+    primary key (report_id, review_id)
+);
+
+create table moderator_report_to_review_comment
+(
+    report_id uuid references moderator_report,
```

Comment:
`report_id` should be also unique separately from `review_comment_id`

Reply:
done